### PR TITLE
Webapp yarn updates

### PR DESCRIPTION
**Pull request by jentfoo**
This PR provides an initial set of updates that hopefully will be safe.  It is probably easiest to consume this PR by reviewing each commit in isolation.  Many of the CVE's addressed are not significant for this project, it is still best to generally keep things up to date.

This PR captures all of the PR's currently open here: https://github.com/gravitational/teleport/pulls?q=is%3Aopen+is%3Apr+author%3Aapp%2Fdependabot+label%3Ajavascript

And will address most of the items listed here (remaining items have major version upgrades that I thought would be best to do in subsequent PR's): https://github.com/gravitational/teleport/security/dependabot?q=is%3Aopen+ecosystem%3Anpm

Additionally the last commit cleans up the `yarn.lock` using this tool: https://www.npmjs.com/package/yarn-deduplicate

I am unsure if this is desired, but it made it much easier for me to parse through the file to understand our current dependent versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,5 +38,13 @@
       "web/packages/teleterm",
       "e/web/**"
     ]
+  },
+  "dependencies": {
```
Comment (zmb3) [inline package.json:42]: This doesn't seem right. These updates should probably go in the nested package.json files.

Comment (jentfoo) [reply package.json:42]: Thank you for mentioning this.  I was a little confused by this also.

This was committed as part of the `yarn upgrade` command, but I noticed the dependabot PR's did not have to do this.  I was hoping someone could explain to me a reliable way to make sure the dependency is updated without making the `package.json` change.

What is the correct way to do this?

**Comment by jentfoo** [comment]
> Looks like `yarn.lock` is not up to date in this PR.

It was, I will update the branch with master when we get closer to being able to merge it.

File: package.json (hunk 1 above)
Comment (jentfoo) [reply package.json]: Thank you @hatched, I removed these changes and applied the `make clean-ui`.  Let me know how this looks now.

**Review by zmb3 — APPROVED** [review]

**Review by hatched — APPROVED** [review]
Looks good, QA was good across oss and e builds.

**Pull request merged**